modules/installation-network-user-infra.adoc

```diff
@@ -114,24 +114,39 @@ for platform containers and provide telemetry data to Red Hat.
 ====
 endif::[]
 
-.Load balancers
+[discrete]
+=== Load balancers
 
-Before you install {product-title}, you must provision two layer-4 load
-balancers. The API requires one load balancer and the default Ingress Controller
-needs the second load balancer to provide ingress to applications.
+Before you install {product-title}, you must provision two load balancers that meet the following requirements:
 
+. *API load balancer*: Provides a common endpoint for users, both human and machine, to interact with and configure the platform. Configure the following conditions:
++
+--
+  ** Layer 4 load balancing only. This can be referred to as Raw TCP, SSL Passthrough, or SSL Bridge mode.
+  ** A stateless load balancing algorithm. The options vary based on the load balancer implementation.
+--
++
+[NOTE]
+====
+Session persistence is not required for the API load balancer to function properly.
+====
++
+Configure the following ports on both the front and back of the load balancers:
++
+.API load balancer
 [cols="2a,5a,2a,2a,2a",options="header"]
```

Comment:
I don't know if we care, but it's possible to get the `x` column to center the X. Also, the `a` in each `cols` allows for full Asciidoc syntax, but as none is being used, it can probably be omitted from `cols`:

```
[cols="2,5,^2,^2,2",options="header"]
```

Works locally.